Document/0x06a-Platform-Overview.md

```diff
@@ -130,13 +130,13 @@ A language.lproj folder exists for each language that the application supports.
 
 ![iOS App Folder Structure](Images/Chapters/0x06a/iOS_project_folder.png)
 
-On a jailbroken device, you can recover the IPA for an installed iOS app with [IPA Installer](https://github.com/autopear/ipainstaller "IPA Installer"). During mobile security assessments, developers often give you the IPA directly. They can send you the actual file or provide access to the development-specific distribution platform they use, e.g., [HockeyApp](https://hockeyapp.net/ "HockeyApp") or [Testflight](https://developer.apple.com/testflight/ "Testflight").
+On a jailbroken device, you can recover the IPA for an installed iOS app using different tools that allow decrypting the main app binary and reconstruct the IPA file. simirarly, on a jailbroken device you can install the IPA file with [IPA Installer](https://github.com/autopear/ipainstaller "IPA Installer"). During mobile security assessments, developers often give you the IPA directly. They can send you the actual file or provide access to the development-specific distribution platform they use, e.g., [HockeyApp](https://hockeyapp.net/ "HockeyApp") or [Testflight](https://developer.apple.com/testflight/ "Testflight").
 
 #### App Structure on the iOS File System
 
-Starting with iOS 8, the way applications are stored on the device changed. Previously, applications were unpacked to a folder in the `/var/mobile/applications/` directory. Note, that since iOS 10, the path has changed to `/private/var/containers/Bundle/Application/` Applications were identified by UUID (Universal Unique Identifier), a 128-bit number. This number was the name of the folder in which the application itself was stored. Static bundle and application data folders are now stored somewhere else. These folders contain information that must be examined closely during application security assessments.
+Starting with iOS 8, the way applications are stored on the device changed. Previously, applications were unpacked to a folder in the `/var/mobile/applications/` directory. Note, that since iOS 10, the path has changed again to `/private/var/containers/Bundle/Application/`. Applications were identified by a UUID (Universal Unique Identifier), a random 128-bit number. This number was the name of the folder in which the application itself was stored. The static app bundle and the application data folders are stored in a different location since iOS 8. These folders contain information that must be examined closely during application security assessments.

```

Comment:
maybe we should not describe it like it was all in the past:
- Applications are identified
- This number i

And can you remove `The static app bundle and the application data folders are stored in a different location since iOS 8. ` as that is already explained at the start of this text block?

Reply:
I have changed the text to clarify pre-iOS8, iOS8 & 9, and after 9.3.x (only slightly, as a deeper and more detailed update would be required).